test/app-server-test.js

```diff
@@ -56,6 +56,25 @@ describe("FastBootAppServer", function() {
       });
   });
 
+  it("executes preFastbootMiddlewares", function() {
+    return runServer('prefastboot-middleware-server')
```

Comment:
We discussed at the meeting this week that the API for this should be to expose pre-FastBoot middleware and post-FastBoot middleware callbacks that yield the Express app itself. This is a more flexible API that is not limited to just middleware, and lets folks use the Express API they're already familiar with instead of learning something custom to the FastBoot App Server.